dev/devicelab/lib/tasks/perf_tests.dart

```diff
@@ -895,144 +910,155 @@ class StartupTest {
       const int iterations = 5;
       final List<Map<String, dynamic>> results = <Map<String, dynamic>>[];
 
-      section('Building application');
-      String? applicationBinaryPath;
-      switch (deviceOperatingSystem) {
-        case DeviceOperatingSystem.android:
-          await flutter(
-            'build',
-            options: <String>[
-              'apk',
-              '-v',
-              '--profile',
-              '--target-platform=android-arm,android-arm64',
-              '--target=$target',
-            ],
-          );
-          applicationBinaryPath = '$testDirectory/build/app/outputs/flutter-apk/app-profile.apk';
-        case DeviceOperatingSystem.androidArm:
-          await flutter(
-            'build',
-            options: <String>[
-              'apk',
-              '-v',
-              '--profile',
-              '--target-platform=android-arm',
-              '--target=$target',
-            ],
-          );
-          applicationBinaryPath = '$testDirectory/build/app/outputs/flutter-apk/app-profile.apk';
-        case DeviceOperatingSystem.androidArm64:
-          await flutter(
-            'build',
-            options: <String>[
-              'apk',
-              '-v',
-              '--profile',
-              '--target-platform=android-arm64',
-              '--target=$target',
-            ],
-          );
-          applicationBinaryPath = '$testDirectory/build/app/outputs/flutter-apk/app-profile.apk';
-        case DeviceOperatingSystem.fake:
-        case DeviceOperatingSystem.fuchsia:
-        case DeviceOperatingSystem.linux:
-          break;
-        case DeviceOperatingSystem.ios:
-        case DeviceOperatingSystem.macos:
-          await flutter(
-            'build',
+      if (enableLazyShaderMode) {
+        _addLazyShaderMode(testDirectory);
+      }
+
+      try {
```

Comment:
It's hard to tell but it looks like this is all the same except for the try block.  Just making sure that's the case.

Reply:
Correct